circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: "stable"
+dependencies:
+  pre:
+    - npm install -g bower
+    - sudo apt-get update; sudo apt-get install -y --only-upgrade google-chrome-stable
```

Comment:
This slows down the build and doesn't do anything. We should just check that the containers are built using Ubuntu 14.04 in the settings and it'll have Chrome 50 by default. Also see ECH for the updated circle.yml